projects/admin/src/app/api/loan-api.service.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2021 RERO
+ * Copyright (C) 2021 UCLouvain
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { AbstractApiService } from './abstract-api.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoanApiService extends AbstractApiService{
+
+  static RESOURCE_TYPE_NAME = 'loans';
+
+  // CONSTRUCTOR ====================================================
+  /**
+   * Constructor
+   * @param _http - HttpClient
+   */
+  constructor(
+    private _http: HttpClient
+  ) {
+    super();
+  }
```

Comment:
Remove this

Reply:
I can't because the parent doesn't (yet) declare the HttpClient